### DISCUSSION TITLE
Is there a way to see which options for the active learning model I selected after starting the review, or even when downloading the results (in oracle mode)?

### DISCUSSION THREAD

**Discussion by brittdev** [asreview/asreview#903] [Q&A] [answered]
Maybe that's already possible, but I can't seem to find it. 

When starting a review in oracle mode, after selecting the settings for the active learning model and finishing the set-up there is no way (that I know) to see what my selected settings were. I can of course write it in the description of my project before starting the review, but it would also be nice to be able to check afterwards which options I selected since I tend to be forgetful at times.

**Comment by SagevdBrand** [comment, marked as answer]
Hi @brittdev,

Excellent question! As far as I know it is not directly visible in the front-end, but there is a workaround.
You can find your project in the hidden file .asreview  (see [here](https://github.com/asreview/asreview/discussions/575#discussioncomment-609575) where to find it)
and then open the kwargs.json file in the project's folder within the .asreview folder.

This will show you the settings of your active learning model.

**Reply by brittdev** [reply]
Ah I see! Thanks for showing me the workaround!

**Reply by Rensvandeschoot** [reply]
In version 1 it will be possible to the settings of the model at any time; follow the progress of Version 1 in this [PR](https://github.com/asreview/asreview/pull/688).
